package exception_handling;

public class Return_Try1 {

	public static void main(String[] args) {
		// Case1 : Return statement in try block but do not have a return statement at the end of method
		
		//m1();
	}
	
//	static int  m1() // Compile time error.
//	 { 
//	   try {
//	      System.out.println("I am in try block");
//	      return 50;
//	   }
//	   catch(Exception e)
//	   {
//	     System.out.println("I am in catch block");
//	   }
//	 // Here, no return statement at the end of method.
//	 }
}
